controllers/csiaddons/reclaimspacecronjob_controller.go

```diff
@@ -54,6 +54,7 @@ var (
 	scheduledTimeAnnotation = csiaddonsv1alpha1.GroupVersion.Group + "/scheduled-at"
 )
 
+//+kubebuilder:rbac:groups=coordination.k8s.io,resources=leases,verbs=get
```

Comment:
```suggestion
```
We don't need to find leader in rscronjob controller too right ?

Reply:
we do, because offline reclaim-space needs to run on the leading controller